### PR TITLE
Shaili contribution 

### PR DESCRIPTION
**Pull request by Jusha25**
# Fixed Issues

- First I fixed the Margin.
- Then I fixed the gap between Mobile menu links.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,23 +13,25 @@
...
+        <div class="container">
+            <div class="logo">
+                <a href="#">Temesgen</a>
+            </div>
+            <div class="main_list">
+                <ul class="nav-menu">
+                    <li><a class="nav-link" href="#portfolio">Portfolio</a></li>
+                    <li><a class="nav-link" href="#about">About</a></li>
+                    <li><a class="nav-link" href="#contact">Contact</a></li>
+                </ul>
```
Comment (leonardodiasb) [inline index.html:26]: - [ ] Your nav menu is not fixed, and your mobile version has some horizontal scroll. This is causing a bad behavior and the user can experience the mobile menu make some buttons not visible(see picture below). Kindly fix that.
- [ ] In Firefox browser I experienced some overlap of the text below the mobile menu, and this is making it hard to read the buttons. Kindly fix that.

Your design:
image
Firefox:
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Jusha25 , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
...
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Please use descriptive names to name your branches. The branch should have the name of the feature or action, not your coding partner.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TemesgenA** [comment]
Fixed the required changes.

**Review by OLIPLICHE — APPROVED** [review]
Hi @Jusha25 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**